pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/producer/PulsarKafkaProducer.java

```diff
@@ -34,6 +34,7 @@
 import java.util.concurrent.TimeoutException;
 import java.util.stream.Collectors;
 
+import com.sun.javaws.exceptions.InvalidArgumentException;
```

Comment:
it seems that this is a wrong import?

Reply:
Removed unused import.
Also added document that merlimat mentioned.